contracts/lib.rs

```diff
@@ -233,6 +233,31 @@ mod open_payroll {
 
     /// implementation of the OpenPayroll contract
     impl OpenPayroll {
+        pub fn default(periodicity: u32, base_payment: Balance) -> Self {
```

Comment:
Maybe implement default trait for the struct instead of creating its own internal function